extensions/amp-addthis/0.1/addthis-utils/rot13.js

```diff
@@ -24,6 +24,7 @@ const CHAR_Z_UPPER = 90;
  *
  * @param {string} input
  * @private
+ * @return {*} TODO: Specify return type
```

Comment:
This would ideally be:
```
 * @param {string} input
 * @return {*} TODO: Specify return type
 * @private
```

It doesn't make sense for the visibility modifier to interrupt the function signature types. Not sure if it is too much work to change for this PR in all the places.

Reply:
Good catch. I went ahead and manually self-reviewed and fixed all instances. Done.